src/libsync/vfs/cfapi/vfs_cfapi.cpp

```diff
@@ -114,7 +114,7 @@ Result<void, QString> VfsCfApi::updateMetadata(const QString &filePath, time_t m
         return cfapi::updatePlaceholderInfo(handle, modtime, size, fileId);
     } else {
         qCWarning(lcCfApi) << "Couldn't update metadata for non existing file" << localPath;
-        return "Couldn't update metadata";
+        return {"Couldn't update metadata"};
```

Comment:
@mgallien Please, space after `{` and before `}` similar to how we have it in the rest of the code (at least in the **cfapiwrapper** we are adding spaces).